### DISCUSSION TITLE
After becaming  an Enterprise Organization some Github actions broken for no aparent reason

### DISCUSSION THREAD

**Discussion by rbnunes** [community/community#47488] [Enterprise]
### Select Topic Area

Question

### Body

Just after becaming and Enterprise Organization I start getting this error on GitHub Action

`Run FirebaseExtended/action-hosting-deploy@v0
  
/home/runner/work/_actions/FirebaseExtended/action-hosting-deploy/v0/bin/action.min.js:3759
                const error = new RequestError(message, status, ***
                              ^

RequestError [HttpError]: Resource not accessible by integration
    at /home/runner/work/_actions/FirebaseExtended/action-hosting-deploy/v0/bin/action.min.js:3759:31`

The error itself don't make much sense to me and it was working just before.

**Comment by gidsg** [comment]
I think it may be related to this https://docs.github.com/en/enterprise-server@3.8/admin/policies/enforcing-policies-for-your-enterprise/enforcing-policies-for-github-actions-in-your-enterprise#configuring-the-default-github_token-permissions so `GITHUB_TOKEN` has changed to have restrictive permissions by default, so you will need to specify granular permissions on your action as described in https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#permissions
